Src/Plugins/Tests.SalModernization/TestData/UpdateSalToCurrentVersion/ExpectedOutputs/SEC105_001.AnalysisAssume.sarif

```diff
@@ -14,80 +14,48 @@
           "rules": [
             {
               "id": "SEC105/001",
```

Comment:
>001 [](http://example.com/codeflow?start=28&length=3)

You should go back to your bucket rules.

RemoveObsoleteOrRedundantAnnotations
RenameLegacyAnnotationsToCurrentVersion
UpdateAnnotationsToCurrentVersion

Or something.

For each of these, use the name of the annotation as a sub-id. Consult the spec for more information. A sub-id is an additional parameter that is in a rule only, it is not valid in a rule definition.

Reply:
changed that. take a look when u can.